### PR TITLE
feat(#54): Add 2 rules: capitalAndLowerCase & noSpaces

### PR DESCRIPTION
**Pull request by saver711**
image

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,251 +1,276 @@
...
+    },
+    match: {
+      valid: value.length > 0 && value === valueAgain,
+      message: messages.match || "Passwords match."
+    },
+    capital: {
+      valid: valueHasCapitalCharacters(value),
+      message: messages.capital || "Password has a capital letter."
+    },
+    lowercase: {
```
Comment (sators) [inline src/index.tsx:117]: Can you add some unit tests to assert that the rules are working correctly?

File: src/index.tsx [hunk 2]
```diff
@@ -1,251 +1,276 @@
...
+      valid: valueHasLowercaseCharacters(value),
+      message: messages.lowercase || "Password has a lowercase letter."
+    },
+    capitalAndLowerCase: {
+      valid:
+        valueHasCapitalCharacters(value) && valueHasLowercaseCharacters(value),
+      message:
+        messages.capitalAndLowerCase ||
+        "Password Contains at least one uppercase and lowercase characters"
```
Comment (sators) [inline src/index.tsx]: Please use lowercase "c" in `contains` (both on this line and in the `noSpaces` rule)

**Review by sators — CHANGES_REQUESTED** [review]
Awesome - thanks for the contribution!

File: src/index.tsx (hunk 1 above)
Comment (saver711) [reply src/index.tsx:117]: @sators Done ✅

File: src/index.tsx (hunk 2 above)
Comment (saver711) [reply src/index.tsx]: @sators Done ✅

**Comment by saver711** [comment]
@sators just mention me when u merge it so that i can use your package

**Review by sators — APPROVED** [review]

**Pull request merged**